demo/demo.js

```diff
@@ -5,8 +5,7 @@ import '../src/style.css';
 
 const repositories = require('./data.json');
 
-const chart = eventDrops();
```

Comment:
I don't like signature of this function. Forcing to pass an empty object to specify d3 looks weird. I would simply pass nothing here.